components/p3a/brave_p3a_uploader.cc

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/p3a/brave_p3a_uploader.h"
+
+#include "base/base64.h"
+#include "net/base/load_flags.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+
+namespace brave {
+
+namespace {
+
+// TODO(iefremov): Provide more details for the traffic annotation.
+net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotation() {
+  return net::DefineNetworkTrafficAnnotation("metrics_report_uma", R"(
+      semantics {
+        sender: "Brave Privacy-Preserving Product Analytics Uploader"
+        description:
+          "Report of anonimized usage statistics. The desciprtion is WIP."
+        trigger:
+          "Reports are automatically generated on startup and at intervals "
+          "while Brave is running."
+        data:
+          "A protocol buffer with anonimized and encrypted usage data."
+        destination: WEBSITE
+      }
+      policy {
+        cookies_allowed: NO
+        setting:
+          "To be implemented."
```

Comment:
this should mention the new pref

Reply:
@iefremov can you help me with the format on this? I know the raw value is `brave.p3a.enabled`

Reply:
this is just a text, something like "Users can enable or disable it in brave://settings, section XX"

Reply:
Updated 👍